src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -0,0 +1,230 @@
+ // Copyright (c) Microsoft. All rights reserved.
+ // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ 
+<%@ WebHandler Language="C#" Class="PullRequestHandler" %>
+
+// This script is called to prepare a local repo for testing
+// ASSUMES: 
+//   - Git is installed on the system to "C:\Program Files\Git\bin\git.exe"
```

Comment:
Related to my [question below](https://github.com/dotnet/wcf/pull/981/files#r58778632), but it needs to be updated either way.